src/main/java/de/caritas/cob/uploadservice/api/facade/EmailNotificationFacade.java

```diff
@@ -36,7 +41,9 @@ public EmailNotificationFacade(
    */
   public void sendEmailNotification(String rcGroupId) {
     emailNotificationHelper.sendEmailNotificationViaUserService(
```

Comment:
Have you missed to include the condition for "multitenancy" or is the member here obsolete?

Reply:
I removed multitenancy property as this is not needed, in case of non-multitenant deployments, it will pass Optional.empty, as tenant parameter therefore we don't need any if condition for the multitenancy check